Sources/AppStoreConnectCLI/Services/Options/ReadBuildOptions.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ReadBuildOptions.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Nafisa Rahman on 1/5/20.
+//
+
+import Foundation
+
+struct ReadBuildOptions {
```

Comment:
We have started putting these inside the Operation. Take a look at: `ListBetaGroupsOperation`